Removed NO MORE update in hide and show stats per #74

andmarek opened the pull request:
Co-authored-by: Eric Marcanio <ericmarcanio@gmail.com>
Co-authored-by: Matthew Phipps <mnphipps98@gmail.com>

Looks like there was no reason to update this.NO_MORE in hideStats() unless hideStats() is a misnomer.

andmarek commented:
fixes #74

samtheprogram commented:
- the NO_MORE prop should be removed, such that the ReceiverConnection manages the StatsHandler itself
- This breaks the Stats dropdown since NO_MORE is no longer triggering ReceiverConnection to open the StatsHandler -- so ReceiverConnection needs to set up StatsHandler regardless when the connection is created.
- Finally, when ReceiverConnection is unmounted, the StatsHandler needs to be destroyed.

andmarek commented:
I deleted the NO_MORE data in Receiver and removed any trace of it.  I find that a bigger issue is that there is no onPeerStateChanged when a user who is viewing a stream clicks on the "2N" at the top and goes back to the home page.  This could root from the issue with the viewers not going down when a user leaves as well.  I believe this is why the stream data keeps getting sent, as the getStats() call only happens when the peer state is connected.

samtheprogram commented on src/components/ReceiverConnection.vue:
```diff
@@ -227,6 +228,7 @@ export default {
         event.iceConnectionState === 'connected' &&
         event.signalingState === 'stable'
       ) {
+        console.log(event.iceConnectionState);
```
This could be removed.

samtheprogram commented on src/views/Receiver.vue:
```diff
@@ -80,6 +80,8 @@ video
 
 #volume-slider
   max-width: 120px
+  @supports (-webkit-touch-callout: none) // iOS volume slider doesn't work, so hide it
```
This should be in a different PR / ticket... was this written in this PR or did it end up in this branch after pulling from master?

samtheprogram commented on src/views/Receiver.vue:
```diff
@@ -217,8 +219,6 @@ export default {
       isPlaying: false,
       theaterMode: false,
       statsVisible: false,
-      volume: window.localStorage.getItem('volume') || 0.5,
```
🤔 this should definitely not been removed

samtheprogram commented on src/views/Receiver.vue:
```diff
@@ -397,21 +394,18 @@ export default {
-      // TODO: Should this be 1024?
-      var k = 1000;
-      var sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+      let k = 1000;
```
These could be const instead of let.

samtheprogram commented:
These changes were rebased and applied in PR #133. Thanks! Closing PR.
